test/run_pass/test_os_network_interfaces.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+var assert = require('assert');
+var os = require('os');
+
+var family = {
+  IPv4: 1,
+  IPv6: 1
+};
+
+var net_info = os.networkInterfaces();
+
+for (var name in net_info) {
+  assert(net_info[name].length);
+  net_info[name].forEach(net => {
+    assert.equal(typeof net.address, 'string');
+    assert.equal(typeof net.netmask, 'string');
+    assert.equal(family[net.family], 1);
+    assert.equal(typeof net.broadcast, 'string');
+    assert.equal(typeof net.mac, 'string');
+  });
+}
```

Comment:
Could you please just move the Node.js [test](https://github.com/nodejs/node/blob/master/test/parallel/test-os.js#L131-L172) into this repository?

Reply:
Yes, I will try it.